src/components/theTimer.css

```diff
@@ -0,0 +1,67 @@
+.timer-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 100%;
+    padding-top: 5%;
+}
+.timer-section{
+    background: rgb(68, 68, 68);
+    width: max-content;
+    padding: 2% 5%;
```

Comment:
preferrably padding can be put in ems rems or px if <10px. % might be an issue in mobile devices

Reply:
done